ls1intum/Artemis
# New Information Messages for Exercise Assessment Dashboard

### Checklist

### Motivation and Context


Currently if no new submissions are available for tutors, they don't see why. This can be confusing. In this PR we add some new informations regarding the state of the assessment process.

### Description

We add a new information popup that tells the assessor exactly why there are no new submissions.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> exams -> assessment dashboard
3. In case there are no new submissions and in case there are locks from other tutors, the information should be displayed correctly. The new information says for example: 'There are 0 out of 1 open assessments that you can correct. 2 submissions are locked by other tutors.'
4. For the second correction round, an additional '1 submission is not yet assessed in the first correction round.' may be displayed in case it is true.

### Test Coverage



exercise-assessment-dashboard: 78.33%
ExerciseResource: 92%
### Screenshots


![image](https://user-images.githubusercontent.com/33342534/112213833-c8c20c80-8c1e-11eb-8250-adf0434be708.png)
